core/exceptions/ExceptionStackTraceDisplay.php

```diff
@@ -68,73 +68,75 @@ protected function displayException(Exception $exception)
      * @throws ReflectionException
      */
     private function developerError(Exception $exception, $msg, $code, $trace_details)
     {
         $time = time();
         return '
-	<div class="ee-error-dev-msg-dv">
-		<p class="ee-error-dev-msg-pg">
-		    '
-            . sprintf(
-                esc_html__('%1$sAn %2$s was thrown!%3$s code: %4$s', 'event_espresso'),
-                '<strong class="ee-error-dev-msg-str">',
-                get_class($exception),
-                '</strong>  &nbsp; <span>',
-                $code . '</span>'
-            )
-            . '<br />
-            <span class="big-text">"' . trim($msg) . '"</span><br/>
-            <a id="display-ee-error-trace-1'
-               . $time
-               . '" class="display-ee-error-trace-lnk small-text" rel="ee-error-trace-1'
-               . $time
-               . '">
-                ' . esc_html__('click to view backtrace and class/method details', 'event_espresso') . '
-            </a><br />
-            '
-            . $exception->getFile()
-            . sprintf(
-                esc_html__('%1$s( line no: %2$s )%3$s', 'event_espresso'),
-                ' &nbsp; <span class="small-text lt-grey-text">',
-                $exception->getLine(),
-                '</span>'
-            )
-            . '
-        </p>
-        <div id="ee-error-trace-1'
-               . $time
-               . '-dv" class="ee-error-trace-dv" style="display: none;">
-            '
-               . $trace_details
-               . $this->classDetails() . '
-        </div>
-    </div>';
+        <div class="ee-error-dev-msg-dv">
+            <p class="ee-error-dev-msg-pg">
+                '
+                . sprintf(
+                    esc_html__('%1$sAn %2$s was thrown!%3$s code: %4$s', 'event_espresso'),
+                    '<strong class="ee-error-dev-msg-str">',
+                    get_class($exception),
+                    '</strong>  &nbsp; <span>',
+                    $code . '</span>'
+                )
+                . '<br />
+                <span class="big-text">"' . trim($msg) . '"</span><br/>
+                <a id="display-ee-error-trace-1'
+                   . $time
+                   . '" class="display-ee-error-trace-lnk small-text" rel="ee-error-trace-1'
+                   . $time
+                   . '">
+                    ' . esc_html__('click to view backtrace and class/method details', 'event_espresso') . '
+                </a><br />
+                '
+                . $exception->getFile()
+                . sprintf(
+                    esc_html__('%1$s( line no: %2$s )%3$s', 'event_espresso'),
+                    ' &nbsp; <span class="small-text lt-grey-text">',
+                    $exception->getLine(),
+                    '</span>'
+                )
+                . '
+            </p>
+            <div id="ee-error-trace-1'
+                   . $time
+                   . '-dv" class="ee-error-trace-dv hidden">
```

Comment:
+ 💯  for wanting to replace `style="display: none;"` with a `hidden` class 

**However**, I can almost guarantee that JS is being used to show and hide that container by adding and removing `style="display: none;"`, most likely via `jQuery.show()` and  `jQuery.hide()` or one of its variants (`.toggleSlide()` etc). 
So clicking on whatever listener is currently set up will most likely attempt to display that container by setting `style="display: block;"`.

That will work fine if the `hidden` class is merely:

```css
.hidden {
	display: none;
}
```

however...  `hidden` is a **VERY** common class name used in plugins and templates so there is an extremely high chance of conflicting with one of those. IF the conflicting `hidden` class is doing something other than the above, then things will not work.

possible solutions:

- namespace classname, ex: `ee-hidden` this is something that should be practiced 100% of the time, but even this could still conflict with another usage of `ee-hidden` somewhere else in the plugin (so search first)

- make classname more specific ex: `ee-error-trace-dv--hidden` BEM is great for this (search "css BEM")

- depending on how the JS works, it may also be necessary to modify that logic to toggle classnames instead of using other jQuery visibility handlers like `.show()`,  `.hide()`, `.toggleSlide()` etc. Using css classnames can also give more control over display.

- some other possible solution that you can think of?

I'm always open to new ideas for solving problems, but just be aware that a seemingly simple change can have a domino affect and cause all kinds of unanticipated problems.

Reply:
ya I've been trying to implement it whenever I can, it's a little bit verbose, but works REALLY well with Sass css